packages/server/modules/core/repositories/users.ts

```diff
@@ -51,16 +51,30 @@ export async function getUsers(
   userIds = isArray(userIds) ? userIds : [userIds]
 
   const q = Users.knex<UserWithOptionalRole[]>().whereIn(Users.col.id, userIds)
+  q.leftJoin(
+    USER_EMAILS_TABLE_NAME,
```

Comment:
At this point I don't see why you shouldn't create and use table helpers similar to what we have for all other tables. You're manually templating column strings which has no type checking whatsoever.

Either use `buildTableHelper()` to create a type-checked helper for the table, or use the approach Gergo showed for type-checked column names (`db<Type>().where('col', val) // 'col' should be type checked). The table helper has nicer DX easily.